example/aes.dart

```diff
@@ -0,0 +1,48 @@
+library example.aes;
+
+import "dart:convert";
+import "dart:typed_data";
+
+import "package:hex/hex.dart";
+import "package:pointycastle/pointycastle.dart";
+
+void main() {
+  // Key must be multiple of block size (16 bytes).
+  var key = new Digest("SHA-256").process(
+      utf8.encode("correct horse battery staple"));
+  // Can be anything.
+  var message = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed d"
+      "o eiusmod tempor incididunt ut labore et dolore magna aliqua.";
+  // The initialization vector must be unique for every message, so it is a
+  // good idea to use a message digest as the IV.
+  // IV must be equal to block size (16 bytes).
+  var iv = new Digest("SHA-256").process(utf8.encode(message)).sublist(0, 16);
```

Comment:
Taking the hash of the plaintext for the IV might not be a good idea, as it leaks information about the plaintext. If the text is encrypted twice the IV will be the same. Might be better to use a PRNG.